Sources/Protocols/CellItem/CollectionViewCellItem.swift

```diff
@@ -83,6 +83,20 @@ extension CollectionViewSiblingCellItem {
             objc_setAssociatedObject(self, &AssociatedKeys.sectionItem, newValue, .OBJC_ASSOCIATION_ASSIGN)
         }
     }
+    
+    public weak var cell: UICollectionViewCell? {
+        get {
+            if let object = objc_getAssociatedObject(self, &AssociatedKeys.cell) as? UICollectionViewCell {
+                return object
+            }
+            printContextWarning("We found out that cell property for \(self) is nil")
+            return nil
+        }
+    }
+    
+    internal func _set(cell: UICollectionViewCell) {
```

Comment:
`Internal` visibility is used by default, it's no needs to use it explicitly. And I guess underscore is not a good symbol in function names, what's about:
```suggestion
    func update(with cell: UICollectionViewCell) {
```